src/NubankClient/Nubank.cs

```diff
@@ -12,20 +12,17 @@ namespace NubankClient
 {
     public class Nubank
     {
+        private const string StatementGraphQl = "{\n   \"query\": \"{\\r    viewer {\\r        savingsAccount {\\r            feed {\\r                id\\r                __typename\\r                title\\r                detail\\r                postDate\\r                ... on TransferInEvent {\\r                    amount\\r                    originAccount {\\r                        name\\r                    }\\r                }\\r                ... on TransferOutEvent {\\r                    amount\\r                    destinationAccount {\\r                        name\\r                    }\\r                }\\r                ... on BarcodePaymentEvent {\\r                    amount\\r                }\\r            }\\r        }\\r    }\\r}\"\n}";
```

Comment:
@natenho I was thinking about a way to make it better, representing this query in a way "more typed" but I don't have much experience with graphQL requests. Did you think about it?

Reply:
To be honest, I'm not very familiar with graphQL as well. In fact, I tried to beautify the string and got some errors, and I was unable to find the reason. You may know that it's a "little hard" to debug against NuBank API (tooo many requessstss) ;-), so I decided to keep that "ugly but at least working" string, if you don't mind.

Reply:
@natenho ok, let's keep this way, in the future we can think about it.